test/configCases/plugins/define-plugin/index.js

```diff
@@ -75,4 +75,8 @@ it("should define process.env.DEFINED_NESTED_KEY", function() {
 });
 it("should define process.env.DEFINED_NESTED_KEY_STRING", function() {
 	if(process.env.DEFINED_NESTED_KEY_STRING !== "string") require("fail");
-})
+});
+it("should assign to process.env", function() {
+	process.env.TEST = "test";
```

Comment:
This doesn't test your case. Please test if your test fails without your changes.

``` js
var sub = "SUB";
OBJECT[SUB].FUNCTION(1);
```

This test should trigger the wrong behavior.

---

But make sure to also test this case:

``` js
function f() {
  throw new Error("should not be called");
}
f // <- no semicolon here
OBJECT;
```